legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/nodes/ExecutionNodeExecutor.java

```diff
@@ -259,7 +260,17 @@ public Result visit(PureExpressionPlatformExecutionNode pureExpressionPlatformEx
 
                 AppliedFunction f = (AppliedFunction) pureExpressionPlatformExecutionNode.pure;
                 SerializationConfig config = f.parameters.size() == 3 ? (SerializationConfig) ((ClassInstance) f.parameters.get(2)).value : null;
-
+                if (config != null && config.dateTimeFormat != null)
+                {
+                    try
+                    {
+                        DateTimeFormatter.ofPattern(config.dateTimeFormat.replaceAll("\"","'"));
```

Comment:
This seems like a check to be done at registration or if possible compilation time, no?

Reply:
So, we did try to have this at compilation time but as of now users have to create new instances of AlloySerializationConfig on text mode (and this is not supported on form mode yet) so the pure model will have the instance as a Simple Function Expression and to validate this particular config setting, we had to assume quite few things which doesn't seem reliable, so adding here

Reply:
We will move this validation to Studio end